scripts/quests/crystalWar/On_Sabbatical.lua

```diff
@@ -0,0 +1,134 @@
+                player:getMainJob() == xi.job.SCH and
+                player:getMainLvl() >= xi.settings.main.AF1_QUEST_LEVEL
+        end,
+
+        [xi.zone.THE_ELDIEME_NECROPOLIS_S] =
+        {
+            ['Erlene'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:progressEvent(18)
+                end
+            },
+
+            onEventFinish =
+            {
+                [18] = function(player, csid, option, npc)
+                    local ID = zones[player:getZoneID()]
+                    if npcUtil.giveKeyItem(player, xi.ki.ULBRECHTS_SEALED_LETTER) then
+                        quest:begin(player)
+                    end
+                end,
+            }
+        },
+    },
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.BASTOK_MARKETS_S] =
+        {
+            ['Gentle_Tiger'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Status') == 0 then
+                        return quest:progressEvent(46)
+                    else
+                        return quest:progressEvent(47)
+                    end
+                end
+            },
+
+            onEventFinish =
+            {
+                [46] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Status', 1)
+                end,
+            }
+        },
+
+        [xi.zone.PASHHOW_MARSHLANDS_S] =
+        {
+            ['Indescript_Markings'] =
+            {
+                onTrigger = function(player, npc)
+                    local ID = zones[player:getZoneID()]
+                    local offset = npc:getID() - ID.npc.INDESCRIPT_MARKINGS_OFFSET
+                    if offset == 0 and quest:getVar(player, 'Status') == 1 then
+                        return quest:progressEvent(2)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [2] = function(player, csid, option, npc)
+                    if npcUtil.giveKeyItem(player, xi.ki.SCHULTZS_SEALED_LETTER) then
+                        quest:setVar(player, 'Status', 2)
+                    end
+                end,
+            }
+        },
+
+        [xi.zone.THE_ELDIEME_NECROPOLIS_S] =
+        {
+            ['Erlene'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Status') ~= 2 then
+                        return quest:progressEvent(19)
+                    else
+                        return quest:progressEvent(20)
+                    end
+                end
+            },
+            onEventFinish =
+            {
+                [20] = function(player, csid, option, npc)
+                    local ID = zones[player:getZoneID()]
+                    if player:getFreeSlotsCount() == 0 then
```

Comment:
This is not necessary, as quest:complete() will fail if the item cannot be obtained.  Can just use the below else condition

Reply:
done